layouts/partials/widgets/about.html

```diff
@@ -16,19 +16,21 @@
         <h2 itemprop="name">{{ $.Site.Params.name }}</h2>
         <h3 itemprop="jobTitle">{{ $.Site.Params.role }}</h3>
         {{ if $.Site.Params.organization }}
-        <h3 itemprop="worksFor">
-          {{ with $.Site.Params.organization_url }}<a href="{{ . }}" target="_blank">{{ end }}
-          {{ $.Site.Params.organization }}
+        <h3 itemprop="worksFor" itemscope itemtype="http://schema.org/Organization">
+          {{ with $.Site.Params.organization_url }}<a itemprop="url" href="{{ . }}" target="_blank">{{ end }}
```

Comment:
I prefer to move the `itemprop` to end of the tag as it's relatively low priority, e.g. `<a href="{{ . }}" target="_blank" itemprop="url">`.